### PR TITLE
fix: ArrayUtils.toString NPE caused by null array element

### PR DESCRIPTION
**Pull request by wangjie-fourth**
Fix https://github.com/arextest/arex-agent-java/issues/481

### CODE REVIEW

**Comment by mr3** [comment]
@wangjie-fourth Thanks for your PR, pelase fix unit tests.

**Comment by wangjie-fourth** [comment]
> @wangjie-fourth Thanks for your PR, please fix unit tests.

@mr3 done. please review again

File: arex-instrumentation-api/src/main/java/io/arex/inst/runtime/util/TypeUtil.java [hunk 1]
```diff
@@ -386,7 +386,11 @@ private static String parameterizedTypeToString(ParameterizedType type) {
     public static String arrayObjectToString(Object[] objects) {
```
Comment (mr3) [inline arex-instrumentation-api/src/main/java/io/arex/inst/runtime/util/TypeUtil.java:386]: Please remove this method and call the `ArrayUtils.toString` method directly, as follows

`ArrayUtils.toString((Object[]) object, o -> o.getClass().getTypeName())`

File: arex-instrumentation/dynamic/arex-dynamic-common/src/test/java/io/arex/inst/dynamic/common/DynamicClassExtractorTest.java [hunk 2]
```diff
@@ -572,6 +572,20 @@ void testProceedingJointPoint() throws Throwable {
         assertEquals("[\"java.lang.String\"]", requestType.get(extractor));
     }
 
+    @Test
+    void testProceedingJointPointParamContainNUll() throws Throwable {
```
Comment (mr3) [inline arex-instrumentation/dynamic/arex-dynamic-common/src/test/java/io/arex/inst/dynamic/common/DynamicClassExtractorTest.java:576]: Please change the unit test about `ArrayUtils.toString` method, there are no changes here, keep the unit test.

**Review by mr3 — APPROVED** [review]

**Review by mr3 — APPROVED** [review]

**Review by YongwuHe — APPROVED** [review]

**Pull request merged**